src/helpers/create-redirect.js

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 export function createRedirect(target) {
   return class Redirect extends React.Component {
     componentDidMount() {
-      window.location.href = target;
+      window.location.href = escape(target);
```

Comment:
Also this!

Reply:
This rule is from https://github.com/mapbox/eslint-config-mapbox/blob/master/xss.js - this adds a layer of protection by escaping the url. This prevent hacks! (Which are unlikely in this environment since we define the `target` but it's good practice.